components/vf-banner-elixir/vf-banner-elixir.js

```diff
@@ -0,0 +1,75 @@
+// vf-banner-elixir
+
+// if you need to import any other components' JS to use here
+// import { vfOthercomponent } from vfImportPrefix + '../vf-other-component/vf-other-component';
+
+ /**
+  * The global function for this component
+  * @example vfBanner-Elixir(scope)
+ * @param {object} [scope] - the html scope to process, optional, defaults to `document`
+  */
+function vfBannerElixir(scope) {
+  /* eslint-disable no-redeclare */
+  var scope = scope || document;
+  /* eslint-enable no-redeclare */
+  const elixirBanner = scope.querySelectorAll("[data-vf-js-banner-elixir]");
+  if (!elixirBanner) {
```

Comment:
non-blocking comment: this will never be null, so unnecessary nullcheck

Reply:
Thanks. Updated. I've been burned by not including this check before — but in this case, yeah, redundant. good catch